src/astro/utils/load.py

```diff
@@ -58,28 +66,82 @@ def load_file_into_dataframe(
         elif filetype == FileType.JSON:
             dataframe = pd.read_json(stream, **kwargs)
         elif filetype == FileType.NDJSON:
-            dataframe = pd.read_json(stream, lines=True, **kwargs)
+            if database is None:
+                raise ValueError(
+                    "database param cannot be None when the file type is Ndjson."
+                )
+            dataframe = flatten_ndjson(
+                normalize_config, stream=stream, database=database
+            )
         elif filetype == FileType.PARQUET:
             dataframe = pd.read_parquet(stream, **kwargs)
         else:
             raise ValueError(f"Unable to load file '{filepath}' of type '{filetype}'")
         return dataframe
 
 
+def flatten_ndjson(
+    normalize_config: Union[None, dict], stream: io.TextIOWrapper, database: Database
+) -> pd.DataFrame:
+
+    normalize_config = normalize_config or {}
+
+    check_ndjson_config_delimiter(database=database, normalize_config=normalize_config)
```

Comment:
Pandas dataframes are happy with having columns that have  `.` (default of JSON normalize), but some databases (BQ and Snowflake) aren't. Would it make sense to convert the column name from `.` to `_` on `load_dataframe_into_sql_table`, which is already aware of the database - considering Astro wants to have different default behaviours per database?